Code/IO/src/sitkImageIOUtilities.h

```diff
@@ -23,32 +23,37 @@
 #include <vector>
 #include <ostream>
 
-namespace itk {
+namespace itk
+{
 
 
 // Forward declaration for pointer
 class ImageIOBase;
 
 
-template<class T>
+template <class T>
 class SmartPointer;
 
-namespace simple::ioutils {
+namespace simple::ioutils
+{
 
 /* Internal method which prints all registered ImageIO names and the
  * file extensions they report as supporting.
  */
-SITKIO_HIDDEN void PrintRegisteredImageIOs(std::ostream & out);
+SITKIO_HIDDEN void
+PrintRegisteredImageIOs(std::ostream & out);
 
 /* Internal method which creates a string list of the names of the
-* ImageIOs registered.
+ * ImageIOs registered.
  */
-SITKIO_HIDDEN std::vector<std::string> GetRegisteredImageIOs();
+SITKIO_HIDDEN std::vector<std::string>
+              GetRegisteredImageIOs();
 
 
-SITKIO_HIDDEN itk::SmartPointer<ImageIOBase> CreateImageIOByName(const std::string & ioname);
+SITKIO_HIDDEN itk::SmartPointer<ImageIOBase>
+              CreateImageIOByName(const std::string & ioname);
 
-}
-}
+} // namespace simple::ioutils
```

Comment:
@dave3d any thoughts on this triggering the spell checker as a failure? Perhaps we should not have this end of namespace comment?